### PR TITLE
Map assert contains value

### PR DESCRIPTION
**Pull request by mchmielarz**
#56 containsValue part

### CODE REVIEW

File: src/main/java/org/assertj/vavr/api/AbstractMapAssert.java [hunk 1]
```diff
@@ -253,6 +253,19 @@ public SELF containsOnlyKeys(KEY... keys) {
         return myself;
     }
 
+    /**
+     * Verifies that the actual map contains the given value.
+     *
+     * @param value the value to look for.
+     * @return {@code this} assertions object
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/api/AbstractMapAssert.java]: `.`

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 2]
```diff
@@ -161,14 +162,14 @@ public static Maps instance() {
...
+     * @param info    contains information about the assertion.
+     * @param actual  the given {@code Map}.
+     * @param entries the given entries.
      * @throws NullPointerException     if the given entries array is {@code null}.
      * @throws AssertionError           if the actual map is {@code null}.
      * @throws IllegalArgumentException if the given entries array is empty.
-     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map
-     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones
+     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map.
+     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones.
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: To greedy ;] This is a single sentence! 😂

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 3]
```diff
@@ -247,12 +248,28 @@ public static Maps instance() {
         if (isNotEmpty(notFound)) throw failures.failure(info, shouldContainValues(actual, notFound.toJavaSet()));
     }
 
+    /**
+     * Verifies that the actual map contain the given value.
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: _contains_

File: src/main/java/org/assertj/vavr/internal/Maps.java [hunk 4]
```diff
@@ -247,12 +248,28 @@ public static Maps instance() {
...
+     *
+     * @param <K>    key type
+     * @param <V>    value type
```
Comment (Opalo) [inline src/main/java/org/assertj/vavr/internal/Maps.java]: `.` and above as well.

File: src/main/java/org/assertj/vavr/internal/Maps.java (hunk 2 above)
Comment (mchmielarz) [reply src/main/java/org/assertj/vavr/internal/Maps.java]: Aaaaagrrh

**Pull request merged**